tests/api/test_meteorological_datum.py

```diff
@@ -0,0 +1,26 @@
+from server.core.constants import METEOROLOGICAL_DATA_URI
+
+def test_get_meteorological_data(test_app):
+    response = test_app.get(METEOROLOGICAL_DATA_URI)
+    assert response.status_code == 200
+
+    meteorological_data = response.json()
+
+    first_record = meteorological_data[0]
+    assert "id" in first_record
+    assert "station_id" in first_record
```

Comment:
Let's also test whether we are able to successfully `JOIN` the associated buoy record.

Reply:
Should the `JOIN` be done in the buoys endpoint then? I could write up a method to retrieve first 10 rows of an `INNER JOIN` between buoy and meteorological_data.

Reply:
I primarily wanted you to test whether it works. If you need some guidance, refer to the coastlines models and corresponding CRUD methods to see an example of it in action.

> Should the JOIN be done in the buoys endpoint then?

Please share your thought process and the problem you're trying to solve by doing that. It'll help me better understand your question.

> I could write up a method to retrieve first 10 rows of an INNER JOIN between buoy and meteorological_data.

To test whether it works or not, that sounds fine. If you want to include a method like as part of the PR, I think we should make that limit flexible. 

It's best to avoid hardcoding values like 10. It can quickly become cumbersome to manage. As an alternative, you could write the method to accept an argument sets the value for limit. This argument could have a default value like 10. And that default value should be defined in our const file and imported there.